ez/ez.go

```diff
@@ -167,6 +168,15 @@ func ConfigFileEnvFlag(ctx context.Context, cfg ConfigWithConfigPath, df Decoder
 	basecfg := d.View().(ConfigWithConfigPath)
 	cfgPath, filepathSet := basecfg.ConfigPath()
 	if !filepathSet {
+		// Since we disabled initial verification earlier, let's verify
+		// specifically given that, without a config file, there's no
+		// opportunity for other values to be introduced into the configuration.
```

Comment:
```suggestion
```suggestion
		// Since we disabled initial verification earlier verify the config explicitly.
		// Without a config file, the sources never get re-stacked, so the `Verify()`
		// method is never run by `dials.Config`.
```
```